DEI.md

```diff
@@ -0,0 +1,50 @@
+# DRAFT
+
+# Bronze DEI Project Statement
+
+### _Include Your Project Name/Logo Here_
+### _Include Awarded Badge Here_
```

Comment:
So at this point of them of creating the DEI.md file, they might have not received a badge. So I think this might be confusing?

Reply:
I think this is a really good point. @germonprez do you think it's ok to take this out and advise folks they can put the badge anywhere (including this file) if they want?

Reply:
I like this. This also reduces complexity in the process. As it is right now, you would need to issue a PR against your DEI.md file to put the badge in. This could be something that is often missed by folks.